Meadowlark-js/backends/meadowlark-opensearch-backend/test/integration/UpdateOpensearch.test.ts

```diff
@@ -32,12 +32,15 @@ const resourceInfo: ResourceInfo = {
   resourceName: 'student',
   isDescriptor: false,
   resourceVersion: '3.3.1-b',
+  allowIdentityUpdates: false,
 };
 
-const id = '1234a-5678b';
-
+const meadowlarkId = '1234a-5678b';
+// const documentUuid = getDocumentUuidForDocument();
+// TEMPORAL: Using meadowlarkId as documentUuid
```

Comment:
Out of curiosity, why is this temporal?

Reply:
Because we are creating a documentUuid just for mongodb, but postgres still uses the meadowlarkId. As we use common types and to maintain compatibility, at this time in postgres the meadowlarkId and documentUuid would be taken as the same value.

The point is that it cannot be changed in this ticket because this has implications for keys and queries.

Reply:
We'll need to change postgresql to use documentUuid as well at some point, right? Can you create a new ticket for that @jleiva-gap and put fix version {{meadowlark-0.4.0}}.